homeassistant/components/mutesync/binary_sensor.py

```diff
@@ -0,0 +1,68 @@
+"""mütesync binary sensor entities."""
+import asyncio
+
+import aiohttp
+import async_timeout
+
+from homeassistant.components.binary_sensor import BinarySensorEntity
+
+from .const import DOMAIN
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up the mütesync button."""
+    client = hass.data[DOMAIN][config_entry.entry_id]
+    async_add_entities([MuteStatus(client)], True)
+
+
+class MuteStatus(BinarySensorEntity):
+    """Class to hold mütesync basic info."""
+
+    def __init__(self, client):
+        """Initialize binary sensor."""
+        self.client = client
+        self._status = None
+
+    @property
+    def name(self):
+        """Return the name of the sensor."""
+        return "Mute Status"
+
+    @property
+    def unique_id(self):
+        """Return the unique ID of the sensor."""
+        if self._status is None:
+            return None
+        return f"{self._status['user-id']}-mute"
```

Comment:
The unique ID must be stable, this made it dynamic. Probably the user ID needs to be part of the config flow and stored in the config entry? That way, we can be sure it is available here.